lib/msf/core/exploit/remote/ldap/server.rb

```diff
@@ -71,22 +70,45 @@ def on_send_response(cli, data)
         cli.write(data)
       end
 
+      # Perform the required tasks after pdu object is processed
+      # Override this method in modules to take flow control
+      #
+      def on_processed_pdu(processed_data)
+        pdu_type = processed_data[:pdu_type]
+        case pdu_type
```

Comment:
Unneeded local variable.
```suggestion
        pdu_type = processed_data[:pdu_type]
        case processed_data[:pdu_type]
```